src/script/helpers.js

```diff
@@ -14,3 +14,29 @@ export function request(path, type, data) {
     body: data
   }).then((res) => res.json());
 }
+export const cart = {
+  get: function () {
+    return JSON.parse(localStorage.getItem(cartKey)) || [];
+  },
+  push: function (product) {
+    let currentProducts = JSON.parse(localStorage.getItem(cartKey)) || [];
```

Comment:
```suggestion
    const currentProducts = JSON.parse(localStorage.getItem(cartKey)) || [];
```
có thay đổi gì đâu mà cần phải let đâu em